README.md

```diff
@@ -69,20 +73,21 @@ load("@rules_python//python:pip.bzl", "pip_repositories")
 pip_repositories()
 ```
 
-To depend on a particular unreleased version (not recommended), you can
-use `git_repository` instead of `http_archive`:
+To depend on a particular unreleased version (not recommended), you can do:
 
 ```python
-load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+rules_python_version = "c8c79aae9aa1b61d199ad03d5fe06338febd0774" # Latest @ 2020-10-15
 
-git_repository(
+http_archive(
```

Comment:
`http_archive` is preferred over `git_repository`. See: https://docs.bazel.build/versions/master/external.html#repository-rules